pytmx/pytmx.py

```diff
@@ -1559,15 +1561,19 @@ def read_points(text):
 
         return self
 
-    def apply_transformations(self):
+    def apply_transformations(self) -> List[Point]:
         """Return all points for object, taking in account rotation."""
         if hasattr(self, "points"):
             return rotate(self.points, self, self.rotation)
         else:
             return rotate(self.as_points, self, self.rotation)
 
     @property
-    def as_points(self):
+    def as_points(
+        self,
+    ) -> List[
```

Comment:
not sure if i am reading your change correctly, but i think the return type should be
```
List[Point, Point, Point, Point]
```
the loop in the code could probably be unrolled too, for clarity, if you want